src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
```

Comment:
The font-family 'Avenir' is used without a generic fallback font. It's a good practice to include a generic font family as a fallback to ensure text is displayed correctly if the custom font fails to load. Consider adding a fallback like 'sans-serif'.